src/components/Email/EmailTemplates/AdminInviteEmail.jsx

```diff
@@ -6,14 +6,14 @@ import { Email, Item } from 'react-html-email';
 import { Image } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 
-import OCHLogo from '../../../assets/OCHlogo.png';
+import OCHLogo from '../../../assets/OCH_Logo_SVG.svg';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'OCHLogo' is defined but never used.